core/server/data/meta/description.js

```diff
@@ -1,9 +1,11 @@
 var _ = require('lodash'),
     settingsCache = require('../../settings/cache');
 
-function getDescription(data, root) {
+function getDescription(data, root, options) {
     var description = '',
+        postSdDescription,
         context = root ? root.context : null,
+        options = options ? options : {},
```

Comment:
You either have to rename it to a new variable or you put the line as first line of the function, because `options` exists already. See e.g. https://github.com/TryGhost/Ghost/blob/master/core/server/data/export/index.js#L54